tests/conftest.py

```diff
@@ -73,14 +51,29 @@ def setUp(self) -> None:
         This is called before each test. If you want to add more for your tests,
         Run `JiraTestCase.setUp(self) in your custom setUp() to obtain these.
         """
-        self.test_manager = JiraTestManager()
+
+        initialized = False
+        try:
+            self.test_manager = JiraTestManager()
+            initialized = self.test_manager.initialized
+        except Exception as e:
+            # pytest with flaky swallows any exceptions re-raised in a try, except
+            # so we log any exceptions for aiding debugging
+            LOGGER.exception(e)
```

Comment:
shouldn't you raise the exception after catching then?

Reply:
that gives also a better test result I'd guess

Reply:
I was having trouble with raised exceptions not actually causing the test to fail, so instead I log the exception information like this and then do an assertion just below, as the assertions do work correctly. Let me know if this is still unclear and if I should change the comment to explain this better!